ledger-tool/src/main.rs

```diff
@@ -767,6 +767,18 @@ fn load_bank_forks(
             "Default accounts path is switched aligning with Blockstore's secondary access: {:?}",
             non_primary_accounts_path
         );
+
```

Comment:
@ryoqun I think you did a lot of this about 2 years ago. Just confirming with you that wiping this directory doesn't break some use case.